### PR TITLE
chore: updating links in workflow doc

### PR DESCRIPTION
**Pull request by aludwiko**
References #1460

### CODE REVIEW

File: docs/src/modules/spring/pages/workflow-entities.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 = Implementing Workflow Entities
 :page-aliases: spring:workflow.adoc
-:sample-url: TODO
+:sample-url: https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples/spring-transfer-workflow
```
Comment (efgpinto) [inline docs/src/modules/spring/pages/workflow-entities.adoc:3]: Is this used somewhere?

Comment (efgpinto) [reply docs/src/modules/spring/pages/workflow-entities.adoc:3]: OK, in that case, I think you can use this var in line `#110` .. see the example on that .adoc

Comment (aludwiko) [reply docs/src/modules/spring/pages/workflow-entities.adoc:3]: Nice, I have no idea why IntelliJ did show me this var usage, anyway, thanks.

File: docs/src/modules/spring/pages/workflow-entities.adoc [hunk 2]
```diff
@@ -107,4 +107,4 @@ IMPORTANT: For simplicity purposes, we are returning the internal state directly
 
 //TODO update the `here` link after merge, otherwise the docs validation complain
 
-A full transfer workflow source code is available here. Follow the `Readme` file to run and test it.
+A full transfer workflow source code is available {sample-url}[here]. Follow the `README` file to run and test it.
```
Comment (efgpinto) [inline docs/src/modules/spring/pages/workflow-entities.adoc:110]: I think usually we force the link to open on a new tab if it's an external link. I find it useful.

```suggestion
A full transfer workflow source code is available {sample-url}[here {tab-icon}, window="new"]. Follow the `README` file to run and test it.
```

**Review by efgpinto — APPROVED** [review]
LGTM, left a nit-picking suggestion.

File: docs/src/modules/spring/pages/workflow-entities.adoc (hunk 1 above)
Comment (efgpinto) [inline docs/src/modules/spring/pages/workflow-entities.adoc:3]: To fix ci..

```suggestion
:sample-url: https://github.com/lightbend/kalix-jvm-sdk/tree/main/samples/spring-transfer-workflow

include::ROOT:partial$include.adoc[]
```

File: docs/src/modules/spring/pages/workflow-entities.adoc (hunk 2 above)
Comment (efgpinto) [reply docs/src/modules/spring/pages/workflow-entities.adoc]: I felt the responsability to fix it after my suggestion broke CI :D

Comment (aludwiko) [reply docs/src/modules/spring/pages/workflow-entities.adoc]: thanks :)

**Pull request merged**